src/test/java/com/ibm/cloud/cloudant/kafka/transforms/predicates/IsDesignDocumentTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright © 2022 IBM Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package com.ibm.cloud.cloudant.kafka.transforms.predicates;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import java.util.Collections;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.errors.DataException;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.junit.Test;
+import com.ibm.cloud.cloudant.kafka.common.CloudantConst;
+
+public class IsDesignDocumentTest {
```

Comment:
Maybe add a test for the `CLOUDANT_KEY_NO_ID` case?

Reply:
will do

Reply:
Done in 5cc3a7c38dede0231e2e3f22c2fbd6c6864341d7